ocha_snap.module

```diff
@@ -12,13 +12,13 @@
  */
 
 // A default snap service endpoint.
-define('OCHA_SNAP_URL', 'http://localhost:8442/snap');
+define('FTS_DATA_SNAP_URL', 'http://snap:8442/snap');
```

Comment:
This shouldn't be part of this PR

Reply:
Yes, I have revered this change. Note, using the url (http://localhost:8442/snap) defined in here does not work on my local. Hence, I had changed this based on Peter's comment https://humanitarian.atlassian.net/browse/HPC-5797?focusedCommentId=99367&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-99367

@cafuego can guide us better here.

Reply:
Just use `OCHA_SNAP_URL`here, and then use `variable_get('ocha_snap_url', OCHA_SNAP_URL )` later in the code. My earlier change request for these names are not valid anymore, as this is not our module and we should respect the naming conventions here and not introduce FTS specific terminology.

Reply:
@cafuego Using the value of `OCHA_SNAP_URL` as `'http://localhost:8442/snap'` does not work as based on your comment https://humanitarian.atlassian.net/browse/HPC-5797?focusedCommentId=99367&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-99367 we have to use `'http://snap:8442/snap'`

I suppose this has to be changed.

Reply:
@cafuego Regarding the change of value for OCHA_SNAP_URL, we could handle this via settings.local.inc or update hook at our end and nothing would be needed to be amended. Thanks for the module and your support!